ansible/roles/dhcpd/tasks/main.yml

```diff
@@ -1,53 +1,71 @@
 ---
-# Copyright (C) 2014-2018 Maciej Delmanowski <drybjed@gmail.com>
-# Copyright (C) 2014-2018 DebOps <https://debops.org/>
-# SPDX-License-Identifier: GPL-3.0-only
+# Copyright (C) 2020 CipherMail B.V. <https://www.ciphermail.com/>
+# Copyright (C) 2020 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
 
-- import_role:
-    name: 'secret'
```

Comment:
It might be useful to keep the `secret` role import so that DHCP keys could be stored there and shared between the `dhcpd` and `bind` roles, when the latter is introduced. Then the users could use lookups to retrieve the keys in both roles with the same contents.

Reply:
Agreed. Is it preferred to import the `secret` role in tasks/main.yml or in the playbook?

Reply:
In the `tasks/main.yml` file, please.